mediathread/main/views.py

```diff
@@ -891,6 +891,7 @@ def create_course(self, form, affil):
 
         # Add the current user as an instructor.
         c.faculty_group.user_set.add(self.request.user)
+        c.add_detail('instructor', self.request.user.profile.fullname)
```

Comment:
Nevermind, that case is only used when the row is an Affil, not a course. Still, it doesn't seem right to store the anonymized name.